tests/_support/AcceptanceTester.php

```diff
@@ -18,9 +17,51 @@
 */
 class AcceptanceTester extends \Codeception\Actor
 {
-    use _generated\AcceptanceTesterActions;
+	use _generated\AcceptanceTesterActions;
+
+	/**
+	 * Custom actions here
+	 */
+
+	/**
+	 * Execute a callback during remote acceptance test executions.
+	 *
+	 * @param callable $function Reference to a static class method.
+	 * @param string $file Optional file path, auto detected from method class otherwise.
+	 * @param bool $rollback_on_shutdown Rollback changes on shutdown.
+	 *
+	 * @throws Exception Throws an exception if passed parameters are not as expected.
+	 *
+	 * @return callable
+	 */
+	public function bootstrapWith( callable $function, string $file = null, bool $rollback_on_shutdown = true ) {
+		$path = __DIR__ . '/../../../../tests/load.php';
```

Comment:
I think in order for this to work with the proposed `--path` option we should use a single standard location in a directory we know exists. `file_put_contents()` won't create directories if they don't exist. Could the path be in `vendor`? e.g.
```suggestion
		$path = __DIR__ . '/../../../../vendor/webdriver-test-load.php';
```